### ISSUE TITLE
automatically download `management-api.yaml`

### ISSUE THREAD

**Issue by ndr-brt** [Think-iT-Labs/edc-connector-client#217]
### Check for existing issues

- [X] Completed

### Describe the feature

currently the openapi documentation about the management api is checked in (it will after #215 will be merged), it would be nice to have a way to have it downloaded automatically when there's a new version.
